dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -1146,9 +1247,7 @@ public Response fixConflicts ( @Context final HttpServletRequest httpServletRequ
 
                 PublishingEndPoint endpoint = APILocator.getPublisherEndPointAPI()
                         .findEndPointById(endpointId);
-                String outputPath = ConfigUtils.getIntegrityPath() + File.separator + endpointId;
-                File bundle = new File(
-                        outputPath + File.separator + INTEGRITY_DATA_TO_FIX_ZIP_FILE_NAME);
+                File bundle = new File(IntegrityUtil.getIntegrityDataFile(endpointId, IntegrityUtil.INTEGRITY_DATA_TO_FIX_ZIP_FILENAME));
```

Comment:
Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)